src/cli/ui/renderer.py

```
@@ -359,8 +335,16 @@ def render_assistant_message(message: AIMessage) -> None:
             parts.append(NewLine())
 
         # Render main content
-        content = Renderer._fix_malformed_code_blocks(content)
-        content = Renderer._escape_html_outside_code_blocks(content)
+        if isinstance(content, str):
+            # Use mdformat for consistent, CommonMark-compliant formatting
+            try:
+                content = mdformat.text(content)
+            except Exception:
+                # Fallback to raw content if mdformat fails
+                pass
```

Ruff (0.14.7)

`try`-`except`-`pass` detected, consider logging the exception

(S110)